docs/topics/wasm/wasm-overview.md

```diff
@@ -0,0 +1,62 @@
+[//]: # (title: Kotlin Wasm)
+
+> Kotlin Wasm is [Experimental](components-stability.md).
+> It may be changed at any time. Use it only for evaluation purposes.
+> 
+> We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+[WebAssembly (Wasm)](https://webassembly.org) is a binary format for a virtual machine that can run in web browsers.
```

Comment:
It's not browser only

Reply:
https://webassembly.org
> WebAssembly (abbreviated Wasm) is a binary instruction format for a stack-based virtual machine. Wasm is designed as a portable compilation target for programming languages, enabling deployment on the web for client and server applications.

Reply:
Also, you can get some inspiration from https://blog.jetbrains.com/kotlin/2023/04/kotlin-1-8-20-released/#wasm

Reply:
Ok, rephrased a bit.